doc/io/postgresql/index.md

```
@@ -0,0 +1,44 @@
+- [DMS] (logical, plugin: wal2json)
+- [asyncpg] (logical, native)
+- [pgbelt]: (logical, plugin: pglogical, uses asyncpg)
+- [psycopg2] (protocol, native)
+- [pypg-cdc] (uses psycopg2, modern pgoutput)
+- [pypgoutput] (uses psycopg2)
+- [python-postgres-cdc] (uses psycopg2, even more modern pgoutput)
+- [wal2json] (logical, plugin)
+
+More resources.
+
+- https://github.com/psycopg/psycopg/issues/71
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)